tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+
+Suite Setup         Set Library Search Order  SeleniumLibrary
+Suite Teardown      End Web Test
+
+
+*** Test Cases ***
+Verify Any OpenShift User Can Spawn JupyterLab Notebooks
+    [Documentation]    Verify that any OpenShift user should also be a RHODS user
+    [Tags]    Sanity
+    ...       Tier1
+    ...       ODS-1164
+    Verify Groups ConfigMap Contains The Expected OCP Groups
+    Verify User Can Spawn A Notebook
+
+
+*** Keywords ***
+Verify Groups ConfigMap Contains The Expected OCP Groups
+    [Documentation]    Verify that ConfigMap contains expected Groups
+    ${version_check} =    Is RHODS Version Greater Or Equal Than    1.8.0
+    IF    ${version_check} == True
+        Verify Config Map Group Contains Expected Values   rhods-groups-config  admin_groups   dedicated-admins
+        Verify Config Map Group Contains Expected Values   rhods-groups-config  allowed_groups   system:authenticated
+    ELSE
+        Verify Config Map Group Contains Expected Values   rhods-groups-config  admin_groups   rhods-admins
+        Verify Config Map Group Contains Expected Values   rhods-groups-config  allowed_groups   rhods-users
```

Comment:
Verify Config Map Group Contains Expected Values  The same as the previous keyword about the name. When you use this keyword that wraps the Oc Get value and use it twice you’re doing two calls to the network. It would be better to do just one call with Oc Get and manage the values retrieved.

Reply:
updated